swig/README.md

```diff
@@ -52,8 +52,8 @@ $ make install # optional
 ## How to regenerate the interfaces
 
 To regenerate the interfaces that have already been created. Simply run
-`make` from the `sunrepo/swig` directory. **This will replace all the
-generated files in `sunrepo/src`.**
+`make all32 all64` from the `sundials/swig` directory.
```

Comment:
Alternately, could this just be `make` or `make all`?

Reply:
See my other comment. For some reason `make` / `make all` only run the `all32` target. I have tried many different tweaks to the makefile with no luck.

Reply:
I figured out how to do this, but it's not as clean.  The issue is that `make` seems to determine the dependency tree based on the target names, without realizing that your `.SETINT32` and `.SETINT64` changes those targets.  The way I got this to work was by duplicating each target name, e.g., instead of one `core` target I have two, `core32` and `core64`.  I then group all of the `32` targets together into `modules32`, and all of the `64` targets together into `modules64`, which are used to create `all32` and `all64`.

While my approach will build both sets of interfaces with a single `make` command, it nearly doubles the size of the `Makefile`.  Moreover, it could be confusing when people add new modules, and is not extensible if we wanted to include more variations (e.g., `single`, `double` and `long double` for `sunrealtype`).  I recommend that we stick with @balos1's current approach, even if it requires `make all32 all64`.